tests/helics/test_files/example_connections6.json

```diff
@@ -0,0 +1,14 @@
+//this should be a valid json file (except comments are not recognized in standard JSON)
+{
+  //example json connection file  connections can be an array of two element string arrays  ["publication", "input"] or a map with "publication" and "target" for a single string or an array strings as "targets"
+  "connections": [
+    {
+      "publication": "pub1",
+      "targets": "inp1"
+    },
+    {
+      "endpoint": "ept1",
+      "target": "ept2"
```

Comment:
This example is using a `targets` field instead of `target` -- is that a typo, or is the entry intended to allow an array such as `["inp1"]`?

I'm wondering if maybe the keys should be `input` instead of `target` for links between a publication and input, and something more descriptive for linking endpoints -- `source_endpoint`/`destination_endpoint`?

Reply:
it is actually testing some flexibility in the reader to handle both "target" and "targets"

Reply:
I like that idea though for "source_endpoint" and "destination_endpoint"  I might add that in a subsequent PR.